modules/react/src/components/export-video/modal-tab-export.js

```diff
@@ -31,10 +32,26 @@ function ExportTab({settings, resolution}) {
               searchable={false}
               onChange={settings.setMediaType}
             />
-            <StyledLabelCell>Resolution</StyledLabelCell>
+            <StyledLabelCell>Ratio</StyledLabelCell>
+            <ItemSelector
+              selectedItems={aspRatio}
+              options={['4:3', '16:9']}
```

Comment:
can we have the list of options as a constant variable?

Reply:
Fixed with latest commit. Good catch!